source/slang/slang-ir-use-uninitialized-values.cpp

```diff
@@ -142,8 +142,16 @@ static bool isDifferentiableFunc(IRInst* func)
 static IRInst* resolveSpecialization(IRSpecialize* spec)
 {
     IRInst* base = spec->getBase();
-    IRGeneric* generic = as<IRGeneric>(base);
-    return findInnerMostGenericReturnVal(generic);
+    while (auto baseSpecialize = as<IRSpecialize>(base))
```

Comment:
We shouldn't need this `resolveSpecialization` at all, just call `getResolvedInstForDecoration()`.